JSDemos/Demos/LoadPanel/Overview/description.md

```diff
@@ -1 +1,16 @@
-The LoadPanel is&nbsp;an&nbsp;overlay UI component notifying the viewer that loading is&nbsp;in&nbsp;progress.
\ No newline at end of file
+The LoadPanel is an overlay component used to notify users that a process is in progress. In this demo, you can see how to initialize and configure the component.
+
+## Apply Indicator and Pane
+
+LoadPanel allows you to display a load indicator and a pane behind the indicator. For this, enable the [showIndicator](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#showIndicator) and [showPane](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#showPane) properties. Uncheck the *With indicator* and *With pane* checkboxes to disable the indicator and pane.
+
+## Shade Background
+
+If you want to apply a background shade, enable the [shading](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#shading) property. You can also specify a color of shade and the container to display it. For this, assign the needed color to the [shadingColor](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#shadingColor) property and set the [position.of](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#position) property to a CSS selector of the container. Uncheck the *With overlay* checkbox to disable the background shade. 
+
+## Show and Hide LoadPanel
+
+You can manage the LoadPanel appearance initially and at runtime. Use the [onShown](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#onShown) and [onHidden](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#onHidden) properties to handle the show and hide events at runtime. Enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#hideOnOutsideClick) property to hide the component, when the user clicks outside of ti. To show and hide LoadPanel initially, use the [visible](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) property.
```

Comment:
> the LoadPanel appearance

When we say "appearance", we mean visual aspects, such as color, shape, etc. 

> Use the [onShown](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#onShown) and [onHidden](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#onHidden) properties to handle the show and hide events at runtime.

These properties are only event handlers. We should say how to show and hide the panel first. Besides, there are also the `onShowing` and `onHiding` event handlers. They allow you to cancel the show or hide events based on a condition.

> To show and hide LoadPanel initially, use the [visible](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) property.

In Angular, Vue, and React, this property is used to control the panel visibility both initially and at runtime. In jQuery, we have the `show()`, `hide()`, and `toggle(showing)` methods.

I suggest the following structure for this paragraph:

- How to control the visibility in code: visible / show() / hide() / toggle(showing)
- How can users hide the component: hideOnOutsideClick
- How can I handle the show and hide events: onShowing / onShown / onHiding / onHidden